.github/workflows/python/review_pr.py

```diff
@@ -187,15 +206,29 @@ def getChanges(patch_details):
     }
 
 
-def validate_row_formatting(line):
-    FORMATTING_ERROR = "Line Format Error: The expected line should be: | `full name` | [git-username](https://github.com/git-username) | dd-month-yyyy | \n"
-    # Regular expression for validating the line format
-    format_re = "\+\|\s*`[A-Za-z]+(\s[A-Za-z]+)*`\s*\|\s*\[[a-zA-Z\d](?:[A-Za-z\d]|-(?=[a-zA-Z\d])){0,38}\]\(https:\/\/github\.com\/[a-zA-Z\d](?:[A-Za-z\d]|-(?=[a-zA-Z\d])){0,38}\)\s*\|\s*[\d]{2}-[a-zA-Z\d]+-[\d]{4}\s*\|"
-    # Regular expression for checking extra spaces at the begining of the line
-    extra_spaces_re = "\+\s+\|\s*`[A-Za-z]+(\s[A-Za-z]+)*`\s*\|\s*\[[a-zA-Z\d](?:[A-Za-z\d]|-(?=[a-zA-Z\d])){0,38}\]\(https:\/\/github\.com\/[a-zA-Z\d](?:[A-Za-z\d]|-(?=[a-zA-Z\d])){0,38}\)\s*\|\s*[\d]{2}-[a-zA-Z\d]+-[\d]{4}\s*\|"
-    # Regular expression for checking single qoutes instead of back ticks
-    single_qoutes_re = "\+\|\s*'[A-Za-z]+(\s[A-Za-z]+)*'\s*\|\s*\[[a-zA-Z\d](?:[A-Za-z\d]|-(?=[a-zA-Z\d])){0,38}\]\(https:\/\/github\.com\/[a-zA-Z\d](?:[A-Za-z\d]|-(?=[a-zA-Z\d])){0,38}\)\s*\|\s*[\d]{2}-[a-zA-Z\d]+-[\d]{4}\s*\|"
-    if re.match(format_re, line):
+def validate_row_formatting(files_updated, line):
+    employer_cla_file = 'employer_contributor_license_agreement.md'
+    FORMATTING_ERROR = ""
+    line_format_re = ""
+    extra_spaces_re = ""
+    single_qoutes_re = ""
+
+    if files_updated[0] == employer_cla_file:
```

Comment:
* files_updated[0] may not always be employer cla file. in which case we have to search for the appropriate entry from the files_updated list.
* Shouldn't we repeat the same check for personal contributor files as well?

Reply:
The actions already has a check to make sure that the user has updated only one file in the pull request. 
```python
def validate_has_only_a_single_file_change(pr_details):
    files_updated = pr_details['files_updated']
    if len(files_updated) != 1 :
        message = '## Error: The pull request should have exactly one file change signing the CLA. \nBut found the following files changed: '
        for file in files_updated:
            message += '\n   * ' + file
        return task_failed(message)
    print('Pass: Pull request has only a single file change.')
```
So files_updated  list always consists of only one element. Will add some additional checks to make sure that the updated file is either employer-cla or personal-cla.

Will add some additional checks to make sure that the file updated is either employer-cla or personal-cla